lib/rack/tracker/google_analytics/google_analytics.rb

```diff
@@ -32,14 +32,12 @@ def tracker
   end
 
   def tracker_options
-    @tracker_options ||= begin
-      tracker_options = {}
+    @tracker_options ||= {}.tap do |tracker_options|
       options.slice(*ALLOWED_TRACKER_OPTIONS).each do |key, value|
         if option_value = value.respond_to?(:call) ? value.call(env) : value
```

Comment:
value.try(:call) || value

will be even shorter :) or is this behaving different?